docs/tutorials/08_quantum_kernel_trainer.ipynb

```diff
@@ -36,21 +36,17 @@
    "source": [
     "# External imports\n",
     "from pylab import cm\n",
-    "import pandas as pd\n",
     "from sklearn import metrics\n",
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
     "\n",
-    "%matplotlib inline\n",
-    "\n",
     "# Qiskit imports\n",
     "from qiskit import QuantumCircuit\n",
     "from qiskit.circuit import ParameterVector\n",
-    "from qiskit_aer import AerSimulator\n",
     "from qiskit.visualization import circuit_drawer\n",
     "from qiskit.algorithms.optimizers import SPSA\n",
     "from qiskit.circuit.library import ZZFeatureMap\n",
-    "from qiskit_machine_learning.kernels import QuantumKernel\n",
+    "from qiskit_machine_learning.kernels import TrainableFidelityQuantumKernel\n",
```

Comment:
In the intro text, step 3 refers to `QuantumKernel` - it now needs to be a `TrainbleKernel` instance right?

> "3. Set up the `QuantumKernel` and `QuantumKernelTrainer` objects\n",

Reply:
Replaced with "Set up an instance of `TrainableKernel` and `QuantumKernelTrainer` objects"